src/RelayClient.ts

```diff
@@ -83,13 +81,20 @@ class RelayClient extends EnvelopingEventEmitter {
 
   private readonly _httpClient: HttpClient;
 
-  constructor(httpClient: HttpClient = new HttpClient()) {
+  constructor(
+    httpClient: HttpClient = new HttpClient(),
+    private _relayServerUrl?: string
+  ) {
     super();
 
     this._envelopingConfig = getEnvelopingConfig();
     this._httpClient = httpClient;
   }
 
+  public getRelayServerUrl() {
+    return this._relayServerUrl;
+  }
+
```

Comment:
I guess that if`_relayServerUrl` isn't provided at construction time, this is undefined, isn't it?

Reply:
Yes, it is.